common/src/jni/main/cpp/conscrypt/native_crypto.cc

```diff
@@ -2186,7 +2186,9 @@ static jbyteArray NativeCrypto_EC_KEY_marshal_curve_name(JNIEnv* env, jclass, jo
     const EC_GROUP* group = fromContextObject<EC_GROUP>(env, groupRef);
     JNI_TRACE("EC_KEY_marshal_curve_name(%p)", group);
     if (group == nullptr) {
+        env->ExceptionClear();
```

Comment:
It would be nice, but seems to be painful in JNI if it's even possible.  Note that prior to this PR, this code path would crash ART so I don't think it's ever being hit outside of this test.